development.ini

```diff
@@ -35,11 +36,13 @@ cache.short_term.expire = 60
 cache.default_term.expire = 300
 cache.long_term.expire = 3600
 
-CELERY_ALWAYS_EAGER = true
-CELERY_ACCEPT_CONTENT = ['json', 'msgpack', 'yaml']
-CELERY_DEFAULT_QUEUE = 'unicore_cms'
-CELERY_RESULT_BACKEND = redis://localhost:6379/0
+ga.profile_id = 'foo'
+
+[celery]
 BROKER_URL = redis://localhost:6379/0
+CELERY_IMPORTS =
+    unicore.google.tasks
```

Comment:
The [worker config in puppet](https://github.com/praekelt/puppet/blob/develop/modules/unicore_cms_frontend/files/celery.production.ini) will also need to be updated.